src/python/bot/startup/health_check_responser.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Health check responser that checks if all processes are running as expected
+   and response to health checks."""
+
+from http.server import BaseHTTPRequestHandler
+from http.server import HTTPServer
+import os
+
+RESPONSER_IP = 'localhost'
+RESPONSER_PORT = 7123
+EXPECTED_PROCESSES = ['run.py', 'run_bot.py']
+
+IS_HEALTHY = 200
+NOT_HEALTHY = 500
+
+
+class RequestHandler(BaseHTTPRequestHandler):
+  """Handler for GET request form the health checker"""
+
+  def _is_healthy(self):
+    """Check if all processes are running as expected"""
+    running_processes = os.popen("ps -eo command").read()
```

Comment:
Let's use psutil here, which is cross platform. See https://github.com/google/clusterfuzz/blob/26ccc7da38a20a5cf43423b2ad98e58344d50577/src/clusterfuzz/_internal/system/process_handler.py#L639 for examples. 

We can just add a helper function into `process_handler` and use it here.